content/Improve your Rails App searches with Elasticsearch - Leandro Dominguez.md

```diff
@@ -0,0 +1,213 @@
+# Improve your Rails Apps searches with Elasticsearch
+
+
+## Introduction
+
+When it comes to apps with a lot of data, some SQL queries take more time than expected, especially when we want to search text through several model attributes. The most known examples are e-commerces, but this kind of complex queries can happen in any other kind of apps like social media.
```

Comment:
I think it would be nice if we added one more example, "like social media or ..."